src/plugins/intel_gna/src/backend/gna_limitations.cpp

```diff
@@ -15,11 +15,14 @@
 #include "gna/gna_config.hpp"
 #include "gna_graph_tools.hpp"
 #include "gna_lib_ver_selector.hpp"
+#include "gna_limitations.hpp"
+#include "gna_plugin_config.hpp"
```

Comment:
why do we need two  includes of "gna_plugin_config.hpp" one in header file and second in source file?